### DISCUSSION TITLE
How do I import the meshes properly?

### DISCUSSION THREAD

**Discussion by yuk75** [BadDogSkyrim/PyNifly#212] [Q&A]
I'm using MO2 if that makes a difference, but I'm loading blender with it and forcing library load so I dunno.

I'm trying to import the vault suit so I can remove everything mid way down the calves and the arm bands but it's not loading and frankly nifscope is not helping me at all. I'm trying to merge the gunmental vault suit with the SEV exosuit so I can have a cool outfit but I can't seem to figure it out. I thought I know how to do this in blender so I'd try your plugin but I'm not loading anything in Blender.

![blendernif](https://github.com/BadDogSkyrim/PyNifly/assets/6256167/540665af-d23d-4aa9-912b-de23e24b5a0e)
![nifscope](https://github.com/BadDogSkyrim/PyNifly/assets/6256167/bbbaad5a-6e90-4d54-b17a-e98d98778cb9)

**Comment by yuk75** [comment]
I managed to save the outfit as an object and import it that way but it won't save as a nif :(
![image](https://github.com/BadDogSkyrim/PyNifly/assets/6256167/abf2db7d-76f8-4268-a536-7e26808af33b)

If I export as an object and try to import to nifscope it shows me nothing when it loads....edit turns out it's a giant piece of model but not the model I made. If I import into outfit studio it's a giant piece of clothing that's miles below itself....
![image](https://github.com/BadDogSkyrim/PyNifly/assets/6256167/35ea888c-a969-433e-b903-3bfe10672785)

**Comment by yuk75** [comment]
Okay I finally have it but there's no bones when I go look at it in bodyslide so not sure what I did. I will keep trying to figure it out :)

**Comment by BadDogSkyrim** [comment]
Ok, so, your first problem seems to be that you're using NifTools' addon, not mine. If you want to try again with pynifly, I might be able to help.

**Reply by yuk75** [reply]
I've had a LOT of trouble following guides....can you help me understand how to do it please?
